lib/index.js

```diff
@@ -106,6 +106,11 @@ Segment.prototype.initialize = function() {
   this.ready();
 
   this.analytics.on('invoke', function(msg) {
+    var options = msg.options();
+    if (options.blockedTraits && msg.obj.traits) {
```

Comment:
Is msg.obj guaranteed to exist? Or does this have the possibility to uncaught?

Reply:
Looks like the facade always sets it but I can add an additional check if you want. 
https://github.com/segmentio/facade/blob/master/lib/facade.js#L46

Reply:
If Facade sets it, I'm alright with it.

Reply:
What if all the traits get blocked?  Would `msg.obj.traits` would be falsy then, and this wouldn't execute?

Reply:
It would be an empty object then, so no.